config.yml

```diff
@@ -0,0 +1,22 @@
+baseURL: "http://localhost:1313/prestashop-specs/"
```

Comment:
Hello, what is the scope of this PR? because this is not production-friendly 😄

Reply:
Fixed :p